src/App.jsx

```diff
@@ -14,33 +16,118 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_NAME = 'name';
+const SORT_FIELD_LENGTH = 'length';
+const ENTER_GOODS = goodsFromServer;
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function filter(arr) {
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] !== ENTER_GOODS[i]) {
+      return false;
+    }
+  }
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  return true;
+}
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+function getPreparedGoods(goods, sortField, reverse) {
```

Comment:
The function name 'getPreparedGoods' is not starting with a verb. As per the checklist, method's name should start with a verb and should describe what the function does. Consider renaming it to something like 'prepareSortedGoods'.